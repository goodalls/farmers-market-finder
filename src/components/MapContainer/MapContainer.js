import React from 'react';

export const MapContainer = () => {
  return (
    <div>
      
    </div>
  );
};

// export default MapContainer;
