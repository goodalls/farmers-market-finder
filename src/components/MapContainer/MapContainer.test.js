import React from 'react';
import { MapContainer } from './MapContainer';
import { shallow } from 'enzyme';

describe('MAP_CONTAINER', () => {
  it('should match the snapshot', () => {
    const wrapper = shallow(<MapContainer google={{maps: {}}} />);
    expect(wrapper).toMatchSnapshot();
  });
});
