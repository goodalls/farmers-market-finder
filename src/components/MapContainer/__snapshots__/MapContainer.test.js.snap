// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MAP_CONTAINER should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <MapContainer
    google={
        Object {
            "maps": Object {},
          }
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <Map
        google={
                Object {
                        "maps": Object {},
                      }
        }
/>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "google": Object {
          "maps": Object {},
        },
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <Map
          google={
                    Object {
                              "maps": Object {},
                            }
          }
/>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "google": Object {
            "maps": Object {},
          },
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactFifteenAdapter {
      "options": Object {
        "supportPrevContextArgumentOfComponentDidUpdate": true,
      },
    },
  },
}
`;
